src/player/update/update_player.c

```diff
@@ -23,6 +24,7 @@ static void (*update_ptr[])(player_t *, scene_t *, window_t *, float) = {
     update_roulade
 };
 
+// TODO: here why ?
```

Comment:
What's this ?

Reply:
ah yeah, just a thought, because the function is empty so i was thinking of this function